File: 60.promedio_datos/promedio_datos.py
# Ejercicio No. 60: Promedio de datos.

# processing
def promediar(lista):
    suma = 0
    cont = 0
    for i in lista:
        suma += i
        cont += 1
    promedio = suma / cont
    return promedio

lista1 = [1,2,3,4,5]
promedio = promediar(lista1)

# output
print(promedio)

File: 60.promedio_datos/PromedioDatos.java
package PromedioDatos;

import javax.swing.JOptionPane;

public class PromedioDatos {
    //Processing
    public static int promediar(int [] array){
        int suma = 0;
        int cont = 0;
        
        for (int i : array) {
            suma += i;
            cont ++;
        }

        int promedio = suma/cont;
        return promedio;
    }
    
    public static void main(String[] args) {
        int[] array = {1,2,3,4,5};
        //Output
        JOptionPane.showMessageDialog(null, promediar(array)); 
        System.exit(0);
    }


    
}
